File: AR Test/AR Test/ScannedLandmark.swift
//
//  ScannedLandmark.swift
//  LookSeeProto
//
//  Created by Christian Barbara on 2/18/26.
//

import Foundation
import SwiftUI

struct ScannedLandmark: Hashable, Codable, Identifiable {
    var id: Int
    var name: String
    var description: String?
    var url: String?
    var category: String
    var confidence: String
    var detectionTime: Double
}

File: Android/LookSee/app/src/main/java/looksee/angelll/com/models/ScannedLandmark.kt
package looksee.angelll.com.models

/** One landmark saved in the user's scan history/library. */
data class ScannedLandmark(
    val id: Int,
    val name: String,
    val description: String? = null,
    val url: String? = null,
    val category: String,
    val confidence: String,
    val detectionTime: Double,
)
